lib/redis.interfaces.ts

```diff
@@ -1,10 +1,8 @@
 import { ModuleMetadata, Type } from '@nestjs/common/interfaces';
-import * as Redis from 'ioredis';
-
-export type Redis = Redis.Redis;
+import { RedisOptions } from 'ioredis';
 
 export interface RedisModuleOptions {
-  config: Redis.RedisOptions & { url?: string };
```

Comment:
There's a chance they had a specific reason to do it this way. `* as Redis` can fix compatibility issues with common module default exports.
Hopefully fixed with the ioredis upgrade.

Reply:
Nice ! 
I guess the only other thing to test would be use with a plain js nest project? 
I think that's a possibility.